src/core/agent/services/ipexCommunicationService.ts

```diff
@@ -171,59 +170,153 @@ class IpexCommunicationService extends AgentService {
   }
 
   @OnlineOnly
-  async offerAcdcFromApply(notification: KeriaNotification, acdc: any) {
-    const msgSaid = notification.a.d as string;
-    const msg = await this.props.signifyClient.exchanges().get(msgSaid);
+  async offerAcdcFromApply(id: string, acdc?: any) {
```

Comment:
Why are we passing ID and fetching from the DB again instead of passing `notification` like before?

And why is acdc optional now?

Reply:
ok nevermind, my mistake we need `linkedGroupRequests`. Comment still remains for acdc being optional though